lib/ui/article/HeroListItem.dart

```diff
@@ -81,10 +82,10 @@ class HeroListItem extends StatelessWidget {
 
   Widget _buildHeroArticleImage(ArticleListItemViewModel articleData,
```

Comment:
let´s name all viewModels. just viewModel, unless there are multiple in the current scope.
```suggestion
  Widget _buildHeroArticleImage(ArticleListItemViewModel viewModel,
```

Reply:
DONE